orchestra/models/core/models.py

```diff
@@ -706,9 +706,11 @@ class Status(ChoicesEnum):
     details = models.TextField(null=True, blank=True)
     section = models.CharField(max_length=255, null=True, blank=True)
     order = models.IntegerField(null=True, blank=True)
+    # DEPRECATED FIELD. [Use status instead.]
```

Comment:
The []'s in the docs were to indicate optional :)
```suggestion
    # DEPRECATED FIELD. Use status instead.
```